accounting/serializers.py

```diff
@@ -16,4 +21,46 @@ class Meta:
             'channel_name',
             'date_created',
             'date_updated'
-        )
\ No newline at end of file
+        )
+
+class ProjectDetailsSerializer(serializers.ModelSerializer):
```

Comment:
ProjectDetail. name classes in singular form as it can contain only 1 object. It's weird to call a singular object in its plural form.

eg.
`project_details = ProjectDetailsSerializer(project)` # weird

this also applies on model classes. the only exception that i can think of is the view classes as there are view classes that returns an array of items (like there is no way of returning just an object) so they are exempted. but most/almost all of the time make your classes singular